.github/workflows/notify_teams.yml

```diff
@@ -139,4 +142,39 @@ jobs:
               "action": "${{ env.action }}"
             }
 
+  send_email_on_review:
+    if: (github.event.action == 'submitted' || github.event.action == 'edited') && github.event.review.state == 'APPROVED'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout actions
+        uses: actions/checkout@v3
+        with:
+          sparse-checkout: |
+            .github/actions
+          path: actions
+          
+      - name: Get PR Data
+        run: |
+          echo "number=${{ github.event.pull_request.number }}" >> $GITHUB_ENV
+          echo "title=${{ github.event.pull_request.title }}" >> $GITHUB_ENV
+          echo "user=${{ github.event.review.user.login }}" >> $GITHUB_ENV
+          echo "status=${{ github.event.review.state }}" >> $GITHUB_ENV
+
+      - name: Send Email on PR Review
+        uses: ./actions/.github/actions/send_email
+        with:
+          username: ${{ secrets.EMAIL_USERNAME }}
```

Comment:
This is the PR submitter's email? Or is it the person who starts the job.

Reply:
neither, it is the email address of the mail server that sends the notifications